### DISCUSSION TITLE
Adding a shortcut to the right side menu of open shell's start menu.

### DISCUSSION THREAD

**Discussion by stevesousa1** [Open-Shell/Open-Shell-Menu#1948] [Q&A] [answered]
How (or is it possible) to add a shortcut to the right side menu of open shell's start menu. 
![image](https://github.com/user-attachments/assets/d6826dbf-2bc4-4cb8-986e-52fb544e00a7)

**Comment by ge0rdi** [comment]
You can customize right side menu in `Settings` -> `Customize Start Menu`.
Right-click on the table, select `Insert New Item`.

**Comment by stevesousa1** [comment]
Yes but then how do I add a shortcut I created to a folder.. I have a shortcut to an important folder I use (currently on my desktop) I want it also on right menu for all users.

**Reply by ge0rdi** [reply, marked as answer]
You can select `Custom` command for newly created item. Then right-click and `Edit Item`.

Here you can choose you existing folder shortcut in `Command` field.
Or select your folder in `Link` field - the item will then auto expand just like other folders in start menu.

**Comment by stevesousa1** [comment]
Thank you I got it added but as other (none technical) users use the computer they do not get the same custom menu (with the added/needed quick link).  Do you know if I can I get the new custom start menu to take as the default (win7) menu for all new users.  So everyone gets the same short cut and look

**Reply by ge0rdi** [reply]
I guess you have created #1953 for that so lets continue there.
